Artesian/Artesian.SDK.Tests/MarketDataQueries.cs

```diff
@@ -159,7 +159,7 @@ public void SearchFacet_SearchFacetAsync()
                     Filters = filterDict,
                     Sorts = new List<string>() { "OriginalTimezone" }
                 };
-                var mdq = mds.SearchFacetAsync(filter).ConfigureAwait(true).GetAwaiter().GetResult();
+                var mdq = mds.SearchFacetAsync(filter: filter, doNotLoadAdditionalInfo: false).ConfigureAwait(true).GetAwaiter().GetResult();
```

Comment:
There are 2 SearchFacetAsync overloads.
- SearchFacetAsync(ArtesianSearchFilter filter, CancellationToken ctk = default)
- SearchFacetAsync(ArtesianSearchFilter filter, bool doNotLoadAdditionalInfo = false, CancellationToken ctk = default)

I will remove the first to fix the ambiguity error. It should not be a breaking change as doNotLoadAdditionalInfo  defaults to false

Reply:
That would break current clients that does 
.SearchFacetAsync(filter, ctk)

Best action is not to make doNotLoadAdditionalInfo have a default, or add an overload with only Filter.